typedjsonrpc/registry.py

```diff
@@ -109,7 +109,7 @@ def _handle_exceptions(self, method, is_notification=False, msg_id=None):
                     debug_url = self._store_traceback()
                 else:
                     debug_url = None
-                new_error = InternalError.from_error(exc, debug_url)
+                new_error = InternalError.from_error(exc, self.json_encoder, debug_url)
```

Comment:
json encoder is still a class. Request 55 will fix it (https://github.com/palantir/typedjsonrpc/pull/55) For now you will get an unexpected type error - encode expects an instance and gets a class.